app/views/project_content_items/_add_flags.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class='row flag-for-review'>
+  <h1>Flag '<%= content_item.title %>' for review</h1>
+
+  <%= form_for content_item, url: update_flags_project_content_item_path(project, content_item), method: :post, html: { class: 'col-md-6', data: { module: 'radio-toggle' } } do |f| %>
+
+    <div class='radio'>
+      <label>
+        <%= f.radio_button :flag, "needs_help" %>
+        I need help tagging this
+      </label>
+    </div>
+
+    <div class='radio-toggle'>
+      <div class='radio'>
+        <label>
+          <%= f.radio_button :flag, "missing_topic", data: { target: 'suggested-tags' } %>
+          There's no relevant topic for this
+        </label>
+      </div>
+
+      <div id='suggested-tags' class='form-group add-top-margin add-left-border'>
+        <%= f.label :suggested_tags, "Suggest a new topic" %>
+        <%= f.text_field :suggested_tags, class: 'form-control input-md-4' %>
```

Comment:
Perhaps this is an unlikely use case but the "Suggested topic" isn't being removed, only hidden: 

If you were to "Flag for review" a content item and suggest a new topic, resolve that flag and then flag it again, the previously suggested topic would still be prefilled in the text input field. It's probably an unlikely scenario, but it poses more a question from a data perspective, as it might have a undesirable effect on what data we have left behind, as the flag isn't being deleted.

Was that intentional or have I misunderstood something along the way?

Reply:
I'm OK with postponing the decision until we've got a better picture of what we're doing with the data :)